animata/card/reminder-scheduler.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from "react";
+
+import { cn } from "@/lib/utils";
+
+const ReminderScheduler: React.FC = () => {
+  const [isRepeating, setIsRepeating] = useState<boolean>(true);
+  const [repeatInterval, setRepeatInterval] = useState<string>("Weekly");
+  const [selectedDays, setSelectedDays] = useState<string[]>(["Mo", "We", "Th"]);
+
+  const daysOfWeek = ["Mo", "Tu", "We", "Th", "Fr", "Sa", "Su"];
+
+  const toggleRepeating = () => {
+    console.log("Toggled");
+    isRepeating ? setSelectedDays(["Th", "Fr", "Su"]) : setSelectedDays(["Mo", "We", "Sa"]);
```

Comment:
you don't need a state for this, simply use a local variable on top `const selectedDays = isRepeating ? ...`